src/lib.rs

```diff
@@ -333,6 +333,93 @@ pub struct ApiError {
     pub error_description: Option<String>,
 }
 
+/// # Example
+///
+/// ```
+/// # extern crate mammut;
+/// # use mammut::StatusesRequest;
+/// let request = StatusesRequest::default()
+///                               .only_media()
+///                               .pinned()
+///                               .since_id("foo");
+/// # assert_eq!(&request.to_querystring()[..], "?only_media=1&pinned=1&since_id=foo");
+/// ```
+#[derive(Clone, Debug, Default)]
+pub struct StatusesRequest<'a> {
```

Comment:
Could you add a `new` method that just calls `Self::default()` and replace the use of default in the example with new?

Reply:
done!